src/main/java/io/github/externschool/planner/service/TeacherServiceImpl.java

```diff
@@ -21,18 +24,17 @@
 @Service
 @Transactional
 public class TeacherServiceImpl implements TeacherService {
-    private final TeacherRepository teacherRepository;
-    private final VerificationKeyRepository keyRepository;
-    private final CourseRepository courseRepository;
+
+    private final   TeacherRepository teacherRepository;
+    private final   VerificationKeyRepository keyRepository;
+    private final   ScheduleService scheduleService;
 
     @Autowired
     public TeacherServiceImpl(final TeacherRepository teacherRepository,
-                              final VerificationKeyRepository keyRepository,
-                              final CourseRepository courseRepository) {
+                              ScheduleEventRepository scheduleEventRepository, final VerificationKeyRepository keyRepository, ScheduleService scheduleService) {
```

Comment:
when you're adding IDE auto generated fragments to the existing code, please keep previous formatting